### PR TITLE
Bump stomp dependency to 1.4.4 and add dev instructions

### PR DESCRIPTION
**Pull request by MikaelSmith**
Stomp 1.4.4 incorporates two bug fixes we've been carrying in
puppet-agent packaging. 1.4 also included a rewrite that broke SSL that
has now been fixed with this release.

Add instructions for running spec and acceptance tests, and setting up a
local environment.

### CODE REVIEW

**Comment by Magisus** [comment]
Those docs are great; after spending just an hour or so reading about the general structure and concept of MCO, they're definitely enough to get me going if I wanted to test something. Thanks for this!

File: README.md [hunk 1]
```diff
@@ -4,8 +4,87 @@ The Marionette Collective aka. mcollective is a framework to build server orches
...
+
+ActiveMQ can later by stopped with `activemq stop`.
+
+Setup MCollective with acceptance config:
+```
+mkdir -p ~/.puppetlabs/etc/mcollective/ssl-clients
+cp acceptance/files/client.* ~/.puppetlabs/etc/mcollective
+cp acceptance/files/server.* ~/.puppetlabs/etc/mcollective
+cp acceptance/files/ca_crt.pem ~/.puppetlabs/etc/mcollective
+cp acceptance/files/client.pem ~/.puppetlabs/etc/mcollective/ssl-clients
```
Comment (Magisus) [inline README.md]: I'm not seeing a `client.pem` file in that directory. Should there be one, or is this incorrect/out-of-date?

Comment (MikaelSmith) [inline README.md]: Ah, that's probably a mistake.

Comment (MikaelSmith) [inline README.md]: Updated.

File: README.md [hunk 2]
```diff
@@ -4,8 +4,87 @@ The Marionette Collective aka. mcollective is a framework to build server orches
...
+mkdir -p ~/.puppetlabs/etc/mcollective/ssl-clients
+cp acceptance/files/client.* ~/.puppetlabs/etc/mcollective
+cp acceptance/files/server.* ~/.puppetlabs/etc/mcollective
+cp acceptance/files/ca_crt.pem ~/.puppetlabs/etc/mcollective
+cp acceptance/files/client.crt ~/.puppetlabs/etc/mcollective/ssl-clients/client.pem
+```
+
+Modify `client.cfg` to work on the local machine:
+* Change the `ssl_server_public`, `ssl_client_private`, `ssl_client_public`
+paths to point to `~/.puppetlabs/etc/mcollective/{server.crt,client.key,client.pem}`.
```
Comment (Magisus) [inline README.md]: I assume this is also supposed to be `client.crt` then. Haven't gotten to the end yet so I haven't tried it.

Comment (MikaelSmith) [inline README.md]: Yup.

Comment (MikaelSmith) [inline README.md]: If you want to figure out the right sed commands to do that, I'd drop them in.

**Comment by Magisus** [comment]
Following the instructions through end to end, I was able to get this up and running! Going to merge so others can do the same.

**Pull request merged**